### ISSUE TITLE
Links in /etc/apt/sources.list doesnt work

### ISSUE THREAD

**Issue by dtometzki** [sipeed/LicheePi4A#2]
Hello together,

the links for /etc/apt/sources.list doesnt work as described in chapter 4. in wiki https://wiki.sipeed.com/hardware/en/lichee/th1520/lpi4a/5_desktop.html

Could you tell me other links or when are the links availabel ?
Best regards
Damian

**Comment by dtometzki** [comment]
hello together,

now apt update works but now i got the message  the public-key is not available NO_PUBKEY 2FB3A9E77911527E

```
root@lpi4a:~# apt update
Get:1 https://mirror.iscas.ac.cn/revyos/revyos-base sid InRelease [9439 B]
Ign:1 https://mirror.iscas.ac.cn/revyos/revyos-base sid InRelease
Get:2 https://mirror.iscas.ac.cn/revyos/revyos-addons revyos-addons InRelease [5802 B]
Get:3 https://mirror.iscas.ac.cn/revyos/revyos-gles revyos-gles InRelease [5794 B]
Ign:2 https://mirror.iscas.ac.cn/revyos/revyos-addons revyos-addons InRelease
Ign:3 https://mirror.iscas.ac.cn/revyos/revyos-gles revyos-gles InRelease
Fetched 21.0 kB in 3s (6243 B/s)
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
12 packages can be upgraded. Run 'apt list --upgradable' to see them.
W: GPG error: https://mirror.iscas.ac.cn/revyos/revyos-base sid InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 2FB3A9E77911527E
W: GPG error: https://mirror.iscas.ac.cn/revyos/revyos-addons revyos-addons InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 2FB3A9E77911527E
W: GPG error: https://mirror.iscas.ac.cn/revyos/revyos-gles revyos-gles InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 2FB3A9E77911527E
root@lpi4a:~# 
```

**Comment by stolk** [comment]
How did you solve this? I am getting this as well.

**Comment by dtometzki** [comment]
Hello stolk,

after some time i checked it. This are only Warnings
```
Ign:1 https://mirror.iscas.ac.cn/revyos/revyos-base sid InRelease
Ign:2 https://mirror.iscas.ac.cn/revyos/revyos-addons revyos-addons InRelease
Ign:3 https://mirror.iscas.ac.cn/revyos/revyos-gles revyos-gles InRelease
Fetched 21.0 kB in 4s (5034 B/s)
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
W: GPG error: https://mirror.iscas.ac.cn/revyos/revyos-base sid InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 2FB3A9E77911527E
W: GPG error: https://mirror.iscas.ac.cn/revyos/revyos-addons revyos-addons InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 2FB3A9E77911527E
W: GPG error: https://mirror.iscas.ac.cn/revyos/revyos-gles revyos-gles InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 2FB3A9E77911527E

```
**W: -**-> Warning it isnt an Error

best regards
Damian
